decidim-dev/app/controllers/concerns/decidim/dev/needs_development_tools.rb

```diff
@@ -17,8 +17,8 @@ module NeedsDevelopmentTools
       def apply_development_tools
         return unless respond_to?(:snippets)
 
-        snippets.add(:head, helpers.stylesheet_pack_tag("decidim_dev"))
-        snippets.add(:head, helpers.javascript_pack_tag("decidim_dev"))
+        snippets.add(:snippet_styles, helpers.stylesheet_pack_tag("decidim_dev"))
```

Comment:
As above, I would rather name these snippet positions more clearly `:head` and `:foot` which tells me better where I am adding these snippets to.

Reply:
Yep, suggestion accepted